### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by nahnahdev**
## Project Requirements

- [ ] Delete all the boilerplate from CRA (text, images, styles).
- [ ] Create a directory called components.
- [ ] Inside components, create a new Calculator.js file.
- [ ] In Calculator.js, create a React component that matches this design:
- [ ] Use class based components

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    const numbers = [];
+    for (let i = 1; i < 10; i += 1) {
+      numbers.push(<button type="button">{i}</button>);
+    }
+    return numbers;
+  }
+
+  render() {
+    return (
+      <div className="mainUI">
```
Comment (brytebee) [inline src/components/Calculator.js:25]: - [ ] **Required:** Great job coming this far in the development of this application but while you have rendered all buttons you have missed the styling required of the component

Your Design | Template
:---: | :---:
image | image

Kindly style the component to match this design.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @NahnahAJ👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- The README look professional.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (BenMukebo) [inline src/App.js:5]: - [OPTIONAL] Great job so far :clap However According to the project requirements The App.js component should also be refactored into class-based. 




```ruby
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

**Review by BenMukebo — APPROVED** [review]
### Hi @NahnahAJ, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- GitFlow correctly used 👍🏻 
- The Calculator.js match the following given design 🥇 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**